cmd/query/app/static_handler.go

```diff
@@ -22,7 +22,9 @@ import (
 	"path/filepath"
 	"regexp"
 	"strings"
+	"sync"
 
+	"github.com/fsnotify/fsnotify"
```

Comment:
this has BSD-3 license, is it ok to include in  CNCF project?

Reply:
@caniszczyk Chris, does CNCF have guidelines on which licenses in the dependencies are OK to use?

Reply:
@caniszczyk ping

Reply:
I think it's ok, we already have many BSD-3 dependencies like golang.org/x/net and gorilla, according to our application https://github.com/cncf/toc/blob/master/proposals/jaeger.adoc